playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -64,6 +64,17 @@ For reference, this is the configuration specified:
         Password: {{ shadowsocks_password.stdout }}
         Encryption Method: {{ shadowsocks_encryption_method }}
 
+For users behind stricter surveillance (esp. experiencing QOS throttling in PRC), further configuration of shadowsocks along with simple-obfs plugin can be carried out via following configuration:
```

Comment:
I think the "behind stricter surveillance" part would be better expressed as "For users on unreliable or hostile networks (esp. experiencing quality-of-service (QOS) throttling) ..."

"further configuration of shadowsocks along with simple-obfs" -> "further configuration of Shadowsocks to use the `simple-obfs` plugin ..."